PHP4/secure-signal-handling-non-compliant.php

```
@@ -0,0 +1,15 @@
+
+<?php
+/* 
+*  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*  SPDX-License-Identifier: Apache-2.0
+*/
+
+//{fact rule=secure-signal-handling@v1.0 defects=1}
+function nonCompliant1($param)  {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$param'. (undefined)

(UnusedFormalParameter)